Source/WebKit/UIProcess/API/glib/WebKitAuthenticationRequest.cpp

```diff
@@ -445,6 +456,8 @@ gboolean webkit_authentication_request_is_retry(WebKitAuthenticationRequest* req
  * @request: a #WebKitAuthenticationRequest
  * @credential: (transfer none) (allow-none): A #WebKitCredential, or %NULL
  *
+ * Authenticate the #WebKitAuthenticationRequest.  
```

Comment:
This line introduces trailing whitespace, let's remove it.

```suggestion
 * Authenticate the #WebKitAuthenticationRequest.
```

Reply:
Ah thank you for spotting these out! I was just doing some simple grep for the trailing space pattern; must have missed some of them. Will comb through these again